specificatie/openapi.yaml

```diff
@@ -90,35 +90,34 @@ paths:
       description: "Het ophalen van de tabelwaarden van een landelijke tabel.<br>Als er op omschrijving of code gezocht wordt worden alleen actuele waarden geretourneerd. <br> Als de parameter <b> inclusief beeindigd </b> met waarde <i> true </i>  wordt meegegeven bij de call worden ook beeindigde waarden geretourneerd."
```

Comment:
In de description staat nog dat met code kan worden gezocht. Ook is de code query parameter niet verwijderd uit de components/parameters